app/src/main/res/layout-sw600dp-land/faq_single_activity.xml

```diff
@@ -5,7 +5,7 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/color_def_white">
+    android:background="@color/component_color_faq_activity_container_background_color">
```

Comment:
In all these colors I think they should be renamed by replacing 

`....faq_acitivty` with `...faq_single_activity`. If you need to create new colors thats fine.

Reply:
Done